README.md

```diff
@@ -1,58 +1,53 @@
 # ParkHangs
 
-ParkHangs is an application that allows users to view details for all official Vancouver Parks. Users are able to get detailed information about a specific park, save favourite parks, create and integrate events to their Google Calendar, as well as write comments about a park and events.
+ParkHangs is an all-in-one platform that allows users interact with all official Vancouver, British Columbia Parks. Users can get detailed information about a specific park, save favourite parks (if they are logged in), create and integrate events to their own calendar application, as well as write comments about parks and events.
 
 **CPSC 436i Group Project**
 
 **Who is it for?**
 
-Our web application is intended for people who want to enjoy the Vancouver public park system. For people who want to organize a pickup game of basketball, or check what facilities are available so they can plan before they go.
+Our web application is intended for people who want to enjoy the Vancouver public park system. For people who want to organize a pickup game of basketball, check what facilities are available so they can plan before they go, see which parks are rating the best, and see what other users have to say about the park. Our application is available to both signed-in and non signed-in users.
```

Comment:
See which parks are **rated** the best and see what other users have to say about the park.